### PR TITLE
chore(feed): set first-tab as default on activity

### PR DESCRIPTION
**Pull request by jalemany1**
Expected behavior when loads 'Activity' view and your first or default tab isn't 'All'.

### CODE REVIEW

File: classes/hypeJunction/Feed/Router.php [hunk 1]
```diff
@@ -15,6 +15,12 @@ public static function handleActivityPage($segments) {
 
 		$page = array_shift($segments);
 
+		if (!$page) {
+			$tabs = Config::getTabs(true);
+			$main_tab = array_pop(array_reverse($tabs));
```
Comment (hypeJunction) [inline classes/hypeJunction/Feed/Router.php:20]: = `array_shift`

Comment (jalemany1) [reply classes/hypeJunction/Feed/Router.php:20]: Yeah!
I read that...(but not consider the array_reverse() cost 😅)

the complexity of array_pop() is O(1). 
the complexity of array_shift() is O(n).

Maybe better reset()

Comment (hypeJunction) [reply classes/hypeJunction/Feed/Router.php:20]: Got a link?

Comment (mrclay) [reply classes/hypeJunction/Feed/Router.php:20]: Micro-optimization that affects readability 👎

Comment (hypeJunction) [reply classes/hypeJunction/Feed/Router.php:20]: I don't think it even optimizes anything. I would think shifting first element of an array is quicker than reading the length and finding the last element.

**Pull request merged**